include/open62541pp/Client.h

```diff
@@ -146,7 +146,7 @@ class Client {
      */
     void runIterate(uint16_t timeoutMilliseconds = 1000);
     /// Run the client's main loop by. This method will block until Client::stop is called.
-    void run();
+    void run(uint16_t timeoutMilliseconds = 1000);
```

Comment:
What is your reason for this change? If we need this, let's make a new PR for this.

Reply:
I experienced a delay when stopping the client and tried to adjust the timeout, though I now realized it means runIterate will be called in a polling-like fashion in very short intervals without it. I'll revert this change, as there seems to be no notifying mechanism for stopping the client in open62541.

Reply:
Reverted the changes.